core/models/imports.go

```diff
@@ -227,6 +229,13 @@ func (b *ContactImportBatch) getOrCreateContacts(ctx context.Context, db Queryer
 				addModifier(modifiers.NewLanguage(lang))
 			}
 		}
+		if !isActive {
```

Comment:
not even the name, not even any other field

Reply:
I think we treat status like any other field. You can export contacts, change a status to blocked, re-import, and we update status to blocked just like updating any other field. It is a little weird they won't be included in the final group but chatted with @ericnewcomer about this and convinced ourselves that's not a big problem and could be explained to the user in the final summary.

Reply:
Well I was thinking status could just be a special field such as the created_on, last_seen_on, that cannot be changed by imports

But now we allow updating he status in the flow and UI, so that is probably fine by imports too